modules/core/src/smithy4s/PassthroughSchematic.scala

```diff
@@ -0,0 +1,70 @@
+package smithy4s
+
+import schematic._
+import schematic.struct.GenericAritySchematic
+import java.util.UUID
+
+class PassthroughSchematic[F[_]](schematic: Schematic[F])
+    extends Schematic[F]
+    with GenericAritySchematic[F] {
```

Comment:
So unfortunately, this `Passthrough` construct should not extend this. We'll have to use the build-time Boilerplate.

Reply:
Is that because the schematics passed in may not be using the generic arity trait so they would rely on the other functions directly?

Reply:
Yes indeed

Reply:
We could just delete the non-generic arity methods in the schematic interface. I think experience shows that we always use the generic one anyway. 

We would however keep the arity-based methods in the syntax package, to create schemas. They would just all delegate to the generic-arity one.

Reply:
Yeah, I am fine with that. I just implemented them all in case we decide to go that route.. I agree though that the generic route seems to be sufficient.